doc/specs/bookmarks.md

```diff
@@ -0,0 +1,595 @@
+# Bookmarks
+
+## Metadata
+
+Author: killasnake0-wcd
+
+
+## Table of contents
+
+1. Motivation
+1. Business rules
+1. Use cases
+
+## Motivation
+
+Provide a system for users to bookmark torrent groups, artists, collages, 
+or requests made on the website
+
+## Business rules
+
+* Users are able to bookmark a torrent group, artist, collage, or request.
```

Comment:
what about individual torrents? I think what a user can bookmark should be generic